android/services/mongodb-remote/src/main/java/com/mongodb/stitch/android/services/mongodb/remote/Sync.java

```diff
@@ -83,6 +83,17 @@ void configure(@NonNull final ConflictHandler<DocumentT> conflictHandler,
    */
   Set<BsonValue> getSyncedIds();
 
+  /**
+   * A document that is paused no longer has remote updates applied to it.
+   * Any local updates to this document cause it to be resumed. An example of pausing a document
+   * is when a conflict is being resolved for that document and the handler throws an exception.
+   *
+   * @param documentId the id of the document to resume syncing
+   * @return true if successfully resumed, false if the document
+   *         could not be found or there was an error resuming
+   */
+  boolean resumeSyncForDocument(@NonNull final BsonValue documentId);
```

Comment:
This is good. In addition to this, we need a way to get the Set of all documents that are frozen